internal/install/recipe_installer.go

```diff
@@ -502,6 +502,10 @@ func (i *RecipeInstaller) executeAndValidate(m *types.DiscoveryManifest, r *type
 }
 
 func (i *RecipeInstaller) executeAndValidateWithProgress(m *types.DiscoveryManifest, r *types.Recipe) (string, error) {
+	if r.PreInstall.Info != "" {
+		fmt.Println(r.PreInstall.Info)
```

Comment:
We might want to pad this with two spaces before the message.  Same for the `Println` call be low.

Reply:
@zlesnr Since most of these will be multi-line strings, should we just do that padding in the source recipe?

Reply:
Basically we need to achieve things like this:

![image](https://user-images.githubusercontent.com/6722433/107802920-f10d4000-6d2f-11eb-8442-8dd7ef11d43e.png)

Reply:
So wondering if we let the CLI provide indentation from the left, or if we just craft all our strings with two spaces of indentation.

Reply:
That's a good point.  We could take it as is and see what the feedback is too.